module/VuFind/tests/unit-tests/src/VuFindTest/ExportTest.php

```diff
@@ -310,13 +311,13 @@ public function testGetBulkUrl(): void
         $serverUrl->expects($this->once())->method('__invoke')
             ->with($this->equalTo('/cart/doExport'))
             ->will($this->returnValue('http://localhost/cart/doExport'));
-        $view = $this->getMockBuilder(\Laminas\View\Renderer\PhpRenderer::class)
+        $renderer = $this->getMockBuilder(\Laminas\View\Renderer\PhpRenderer::class)
```

Comment:
While you're changing the line anyway, you could simplify this to `$renderer = $this->createMock(PhpRenderer::class);` and delete the next two lines.

Reply:
Done!